src/SDKs/CognitiveServices/dataPlane/Vision/Vision/Generated/Face/Models/UpdatePersonFaceRequest.cs

```diff
@@ -17,24 +17,22 @@ namespace Microsoft.Azure.CognitiveServices.Vision.Face.Models
     /// <summary>
     /// Request to update person face data.
     /// </summary>
-    public partial class UpdatePersonFaceDataRequest
+    public partial class UpdatePersonFaceRequest
     {
         /// <summary>
-        /// Initializes a new instance of the UpdatePersonFaceDataRequest
-        /// class.
+        /// Initializes a new instance of the UpdatePersonFaceRequest class.
         /// </summary>
-        public UpdatePersonFaceDataRequest()
+        public UpdatePersonFaceRequest()
         {
             CustomInit();
         }
 
         /// <summary>
-        /// Initializes a new instance of the UpdatePersonFaceDataRequest
-        /// class.
+        /// Initializes a new instance of the UpdatePersonFaceRequest class.
         /// </summary>
         /// <param name="userData">User-provided data attached to the face. The
-        /// size limit is 1KB</param>
-        public UpdatePersonFaceDataRequest(string userData = default(string))
+        /// size limit is 1KB.</param>
+        public UpdatePersonFaceRequest(string userData = default(string))
```

Comment:
UserDataContract. As we will use it later in LargeFaceList Face - Update and LargePersonGroup Person - UpdateFace.

Reply:
Consider to fix this later

Reply:
resolved